WooCommerce/Classes/ViewRelated/Inbox/InboxNoteRow.swift

```diff
@@ -19,10 +19,19 @@ struct InboxNoteRow: View {
                     .foregroundColor(Color(Constants.dateTextColor))
 
                 VStack(alignment: .leading, spacing: Constants.verticalSpacing) {
-                    // Title.
+
+
+                    // Title Unread (unactioned)
+                    Text(viewModel.title)
+                        .headlineStyle()
+                        .fixedSize(horizontal: false, vertical: true)
+                        .renderedIf(!viewModel.isRead)
+
+                    // Title Read (actioned)
                     Text(viewModel.title)
                         .bodyStyle()
                         .fixedSize(horizontal: false, vertical: true)
+                        .renderedIf(viewModel.isRead)
```

Comment:
I think we have a custom modifier `if` to use for conditional cases. There's no corresponding `else` though, so if we use it we'll have to write it this way:
```suggestion
                    Text(viewModel.title)
                        .if(viewModel.isRead) { $0.bodyStyle() }
                        .if(!viewModel.isRead) { $0.headlineStyle() }
                        .fixedSize(horizontal: false, vertical: true)
```

Reply:
TIL 👍 done here eed7ee4c7455662ff2a29157117d633ddc306f1b